src/components/WorkflowForms/NewClientSignup/GatherInfo/i18n-strings.js

```
@@ -0,0 +1,188 @@
+    "reportNumberRequired": "El número de reporte policial es obligatorio cuando existe un reporte policial",
+    "agencyRequired": "La agencia es obligatoria cuando existe un reporte policial",
+    
+    // PassengersSection
+    "passengersSectionTitle": "Pasajeros Adicionales",
+    "noPassengers": "No se han agregado pasajeros adicionales.",
+    "newPassenger": "Nuevo Pasajero",
+    "name": "Nombre",
+    "phone": "Teléfono",
+    "savePassenger": "Guardar Pasajero",
+    "cancel": "Cancelar",
+    "fileUploadError": "Error al subir el archivo",
```

ESLint

[error] Duplicate key 'fileUploadError'.

(no-dupe-keys)